app/src/main/kotlin/com/metrolist/music/playback/MusicService.kt

```
@@ -810,6 +836,74 @@ class MusicService :
+                    matrixUpdateJob = null
+                }
+
+                for (client in matrixRpcClients) {
+                    client.close()
+                }
+                matrixRpcClients.clear()
+
+                if (matrixEnabled) {
+                    val accounts = try {
+                        matrixJsonConfig.decodeFromString<List<MatrixAccount>>(accountsJson)
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)